packages/atomic/src/pages/examples/standalone.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```

Comment:
Test it out! It works pretty well, run dev & http://localhost:3333/examples/standalone.html